app/src/components/custom/forms/RequestForm.tsx

```diff
@@ -82,6 +87,29 @@ const RequestForm = (props: RequestFormProps) => {
     setAnchorEl(null);
   };
 
+  const handleDelete = async () => {
+    try {
+      const axiosReqConfig = {
+        url: `${Constants.BACKEND_URL}/api/requests/${reimbursementRequest?._id}`,
+        method: `patch`,
+        headers: {
+          Authorization : `Bearer ${authState.accessToken}`
+        },
+        data: {
+          state: RequestStates.DELETED,
+          isAdmin: isAdmin
```

Comment:
Where is `isAdmin` defined?

Reply:
Passed in as a prop. Could change it now that I'm importing auth stuff here anyway.